src/test/rustdoc-ui/intra-doc/email-address-localhost.rs

```diff
@@ -0,0 +1,4 @@
+#![deny(warnings)]
+
+//! Email me at <hello@localhost>.
+//~^ ERROR unknown disambiguator `hello`
```

Comment:
Can you also test that dots don't have warnings?
```suggestion
//! Email me at <hello@localhost>.
//~^ ERROR unknown disambiguator `hello`

//! This link should *not* warn: <me@example.com>
```